.storybook/main.ts

```diff
@@ -44,7 +44,7 @@ const config: StorybookConfig = {
 			to: "/static",
 		},
 	],
-	stories: [relativePath("..", "src", "**", "*.stories.@(js|jsx|ts|tsx|mdx)")],
+	stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
```

Comment:
you _don't_ want the `mdx` stories to match?

Reply:
There's a tendency in the Storybook examples that the straight up docs pages don't include the `.stories` naming. Or something.

Reply:
Also copypasta.